services/CollectionService/.gitignore

```diff
@@ -1 +1,3 @@
 vendor/
+composer.lock
+composer.phar
```

Comment:
On second thought, maybe I'm being too biased.  Let's try it the 'correct' way.  But if it turns out to be a serious PITA I'll issue a PR to get rid of them.